taiga/auth/signals.py

```diff
@@ -8,4 +8,4 @@
 import django.dispatch
 
 
-user_registered = django.dispatch.Signal(providing_args=["user"])
+user_registered = django.dispatch.Signal() # providing_args=["user"]
```

Comment:
why leaving the comment?

Reply:
It is pure documentation to implement the signal launcher. Is a warning

https://github.com/django/django/blob/stable/3.2.x/django/dispatch/dispatcher.py#L38-L45